pkg/mount/scuzz/control.go

```diff
@@ -13,12 +13,21 @@ import (
 // DefaultTimeout is the default timeout for disk operations.
 const DefaultTimeout time.Duration = 15 * time.Second
 
+const (
+	_SECURITY_SUPPORTED     = 0x1
+	_SECURITY_ENABLED       = 0x2
+	_SECURITY_LOCKED        = 0x4
+	_SECURITY_FROZEN        = 0x8
+	_SECURITY_COUNT_EXPIRED = 0x10
+	_SECURITY_LEVEL_MAX     = 0x100
```

Comment:
Why do these constants start with an underscore? Can they be named Go-style "SecurityLevelMax"?